blocks/library/cover-image/index.js

```diff
@@ -146,8 +151,9 @@ registerBlockType( 'core/cover-image', {
 
 		if ( ! url ) {
 			const uploadButtonProps = { isLarge: true };
-			const icon = title ? undefined : 'format-image';
-			const label = title ? (
+			const hasTitle = ! isEmpty( title );
```

Comment:
Was there a need for pulling in this dependency which wasn't satisfied by a simple truthiness check?

```js
const hasTitle = !! title;
```

Reply:
Hi @aduth, both of this versions don't handle the empty array "[]", and title an empty array of children's. My initial version was title && ( ! isArray( title ) || title.length ); which handle that case too, but I got a suggestion that isEmpty would be a simpler approach and I also think the code is simpler with it.

Reply:
Oops! I overlooked the original review comment. I think `isEmpty` is totally fine in that case; if anything, maybe an inline comment clarifying that we could expect the array.